src/main/resources/org/sagebionetworks/web/client/view/ChangeUsernameViewImpl.ui.xml

```diff
@@ -17,20 +14,16 @@
 							carefully choose yours.
             </h4>
             <g:TextBox ui:field="username" addStyleNames="form-control" />
-            <b:Row>
-              <b:Column size="XS_9">
-                <g:SimplePanel
-                  ui:field="errorContainer"
-                  addStyleNames="left margin-top-10"
-                />
-              </b:Column>
-              <b:Column size="XS_3">
-                <g:Button
-                  ui:field="changeUsernameButton"
-                  styleName="right btn btn-primary btn-lg margin-top-10"
-                />
-              </b:Column>
-            </b:Row>
+            <g:Button
+              ui:field="changeUsernameButton"
+              styleName="right btn btn-primary btn-lg margin-top-10"
+            />
+
+            <g:SimplePanel
+              ui:field="errorContainer"
+              addStyleNames="left margin-top-10"
+              width="100%"
+            />
```

Comment:
I had no idea it was possible for a user to have no username, and that we have UI for such a case. The grid layout was weird to begin with, so I just removed it.